src/content/en/updates/2018/09/nic69.md

```diff
@@ -0,0 +1,325 @@
+where you want the user to scroll to a specific point.
+
+<div class="clearfix"></div>
+
+For an image carousel, I’d add `scroll-snap-type: x mandatory;` to the
+scroll container, and `scroll-snap-align: center;` to each image. Then, as the
+user scrolls through the carousel, each image will be smoothly scrolled into
+the perfect position.
+
+```
+#gallery {
+  scroll-snap-type: x mandatory;
+  overflow-x: scroll;
+  display: flex;
+}
+
+#gallery img {
+   scroll-snap-align: center;
+}
+```
+
+CSS Scroll Snapping works well, even when the snap targets have different
+sizes or when they are larger than the scroller! Check out the post
+[Well-Controlled Scrolling with CSS Scroll Snap](/web/updates/2018/07/css-scroll-snap)
+for more details and samples you can try!
+
+
+<div class="clearfix"></div>
+
+## Display cutouts (aka notches) {: #notch }
+
+<figure class="attempt-right">
+  <a href="/web/updates/images/2018/09/notch-extra-margin.png">
+    <img src="/web/updates/images/2018/09/notch-extra-margin.png"
+         alt="mobile phone with display cutout">
+  </a>
+  <figcaption>
+    Browsers add some extra margin on a mobile device with display cutout to
+    prevent content from being covered by the cutout.
+  </figcaption>
+</figure>
+
+
+There are an
+[increasing number of mobile devices](https://www.google.com/search?q=what+phones+have+a+notch)
+being released with a display cutout, sometimes called a notch. To deal with
+that, browsers add a little bit of extra margin to your page so the content
+isn’t obscured by the notch.
+
+**But what if you want to use that space?**
+
+With CSS environment variables and the
+[`viewport-fit`](https://www.w3.org/TR/css-round-display-1/#viewport-fit-descriptor)
+meta tag, now you can. For example, to tell the browser to expand into the
+display cutout area, set the `viewport-fit` property, in the `viewport` meta
+tag to `cover`.
+
+<div class="clearfix"></div>
+
+```
+<meta name='viewport' content='initial-scale=1, viewport-fit=cover'>
+```
+
+You can then use the `safe-area-inset-*`
+[CSS environment variables](https://drafts.csswg.org/css-env-1/) to layout
+your content.
+
+```
+.content {
+  padding: 16px;
+  padding-left: env(safe-area-inset-left);
+  padding-right: env(safe-area-inset-right);
+}
+```
+
+<aside class="key-point">
+  <b>Remember:</b> It’s important to test this well; you're telling the
+  browser it’s OK if some content gets hidden behind the display cutout.
+  And don’t forget to test in both portrait and landscape modes.
+</aside>
+
+
+There’s a great post on the WebKit blog about
+[Designing Websites for iPhone X](https://webkit.org/blog/7929/designing-websites-for-iphone-x/),
+or check out the
+[explainer](https://docs.google.com/document/d/1lbZi18_5cMlLOphpFqTbuI4B0YGykQvvtRbw6j67UyE/preview)
+for more details.
+
+
+<div class="clearfix"></div>
+
+
+## Web Locks API {: #web-locks }
+
+The [Web Locks API](https://github.com/WICG/web-locks) allows you to
```

Comment:
Very atypically, I've already documented this on MDN. https://developer.mozilla.org/en-US/docs/Web/API/Web_Locks_API